src/main/java/org/jenkinsci/plugins/workflow/support/steps/WorkspaceStepExecution.java

```diff
@@ -77,9 +77,9 @@ public boolean start() throws Exception {
     private static final class ExpanderImpl extends EnvironmentExpander {
         private static final long serialVersionUID = 1;
         private final Map<String,String> override;
-        private ExpanderImpl(String var, String value) {
+        private ExpanderImpl(String path) {
             this.override = new HashMap<>();
-            this.override.put(var, value);
+            this.override.put("WORKSPACE", path);
```

Comment:
What I meant was that you do not need to store a whole `Map` in the serial form—just one `String`.

Not a big deal, but cannot easily be changed once introduced.